src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -144,7 +144,7 @@ def escapeForCommand(string):
 # and is primarily used for arbitrary strings (e.g. filenames, paths)
 # that might not be valid unicode to begin with.
 def escape(string):
-    if isinstance(string, basestring):
+    if isinstance(string, str):
```

Comment:
Neat, I didn't realize Django shipped with a copy of `six`!  We've tried to keep from requiring Django in archivematicaCommon however, and when we do the Py3 conversion we'll probably just use `six` as a dependency instead of using Django's.